wiki/quest-modding.md

```diff
@@ -86,6 +86,10 @@ follow the steps below:
 
 ### SideQuest
 
+:::tip Mods Before Friday
+No need for SideQuest if using Mods Before Friday
+:::
+
 This is only required if you mod with a PC (Windows / Mac / Linux). Go to [SideQuest's Download Page](https://sidequestvr.com/setup-howto)
```

Comment:
Should revise this sentence in addition to not conflict as a PC is still required to use MBF

Reply:
A PC is not required to use MBF - it will run fine on an Android phone with an up-to-date browser.